crates/sui-framework/packages/sui-framework/tests/display_tests.move

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::display_tests {
```

Comment:
Should this be
```suggestion
// Copyright (c) Mysten Labs, Inc.
// SPDX-License-Identifier: Apache-2.0

#[test_only]
module sui::display_tests;
```

Reply:
I haven't touched tests yet! So this is intentional.